### ISSUE TITLE
when training for NER task，appears  warning error

### ISSUE THREAD

**Issue by cjymz886** [BITLsy/Transformer_tf2.0#1]
WARNING:tensorflow:Gradients do not exist for variables ['ner_transformer/encoder_layer/multi_head_attention/dense/kernel:0', 'ner_transformer/encoder_layer/multi_head_attention/dense/bias:0', 'ner_transformer/encoder_layer/multi_head_attention/dense_1/kernel:0', 'ner_transformer/encoder_layer/multi_head_attention/dense_1/bias:0', 'ner_transformer/encoder_layer/multi_head_attention/dense_2/kernel:0', 'ner_transformer/encoder_layer/multi_head_attention/dense_2/bias:0', 'ner_transformer/encoder_layer/multi_head_attention/dense_3/kernel:0', 'ner_transformer/encoder_layer/multi_head_attention/dense_3/bias:0', 'ner_transformer/encoder_layer/sequential/dense_4/kernel:0', 'ner_transformer/encoder_layer/sequential/dense_4/bias:0', 'ner_transformer/encoder_layer/sequential/dense_5/kernel:0', 'ner_transformer/encoder_layer/sequential/dense_5/bias:0', 'ner_transformer/encoder_layer/layer_normalization/gamma:0', 'ner_transformer/encoder_layer/layer_normalization/beta:0', 'ner_transformer/encoder_layer/layer_normalization_1/gamma:0', 'ner_transformer/encoder_layer/layer_normalization_1/beta:0'] when minimizing the loss.
